### DISCUSSION TITLE
[Projects Beta] Quickview for issue / PR in table view

### DISCUSSION THREAD

**Discussion by redcatbear** [community/community#7627] [Projects and Issues] [answered]
As a product owner,
I want to be able to see the description of an issue or PR in the table view,
so that I can rank the table items according to their content.

Either a hover box with the content or a side bar like in the board view would fill this need.

**Comment by tiina303** [comment]
Just to extend the request for draft items I'd love to see the description when hovering.

**Comment by redcatbear** [comment, marked as answer]
This has meanwhile been implemented by the GitHub team. So it is solved.
